gameplay_page.html

```diff
@@ -0,0 +1,221 @@
+            </p>
+        </div>
+    </section>
+    
+
+    <!-- Multiplayer Info -->
+    <section class="multiplayer">
+        <div class="container bg-gradient pt-2">
+            <h2 class="underlined">Multiplayer</h2>
+            <p>Perfect Dark features a multiplayer mode where up to four players and eight
+                computer-controlled
+                bots can compete against each other in different arenas. A split-screen is used for multiple
+                players. Players start a game unarmed and with a certain amount of health. Weapons and
+                ammunition
+                are placed around the arena in preset positions. Once a player is killed, they are
+                regenerated
+                unarmed elsewhere in the arena. The objective of each game is determined by the scenario
+                being
+                played. Scenarios range from the traditional deathmatch mode, where players score points by
+                killing opponents, to objective-based modes such as Capture the Flag and King of the
+                Hill.[14]
+                Other scenarios include Hold the Briefcase, where players must take a briefcase and survive
+                with
+                it for as long as possible, and Hacker Central, a game type where players score points by
+                hacking
+                a computer system using a data uplink device.
+            </p>
+        </div>
+    </section>
+    <!-- Additional Images-->
+    <section class="row text-center p-3">
+        <div class="col col-xxl-12 p-3 img">
+            <img src="pd__1.jpg" class="imageMain" alt="Gameplay Image 1">
+        </div>
+        <div class="col col-xxl-12 p-3">
+            <img src="pd__2.jpeg" class="imageMain" alt="Gameplay Image 2">
+        </div>
+        <div class="col col-xxl-12 p-3">
+            <img src="pd__3.jpg" class="imageMain" alt="Gameplay Image 3">
+        </div>
+
+    </section>
+
+    <footer>
+        <div class="container py-3 toplined pb-5">
+            <div class="row">
+                <div class="col-12 text-center">
+                    <p class="footer-content">
+                        <a class="px-2 text-light" href="#pdlogo">Back to Top</a> |
+                        <a class="px-2 text-light" href="#">Contact Us</a>
+                    </p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-12 text-center">
+                    <p class="footer-copyright">© Danyl Kecha, Aaron Ahmed, Anas Saad, Marylou Das Chagas E Silva. All
+                        Rights Reserved.</p>
+                </div>
+            </div>
+        </div>
+    </footer>
+</body>
+
+
+<!-- Option 1: Bootstrap Bundle with Popper -->
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"crossorigin="anonymous">
+</script>
```

Comment:
Script tag should be added to the end of the body (INSIDE OF IT)

Moreover, please, add the space between attributes in script tag (there should be space before crossorigin attribute):
`crossorigin="anonymous" `

Reply:
Thanks, I just fixed those errors